src/Native/Windows/probe-win.ps1

```diff
@@ -35,20 +35,31 @@ function GetCMakeInfo($regKey)
 function LocateCMake
 {
   $errorMsg = "CMake is a pre-requisite to build this repository but it was not found on the path. Please install CMake from http://www.cmake.org/download/ and ensure it is on your path."
-  $inPathPath = (get-command cmake.exe -ErrorAction SilentlyContinue).Path
+  $inPathPath = (get-command cmake.exe -ErrorAction SilentlyContinue)
   if ($inPathPath -ne $null) {
-    return $inPathPath
+    # Resolve the first version of CMake if multiple commands are found
```

Comment:
This change is simply to pick up a similar change in dotnet/coreclr's version (at src\pal\tools\probe-win.ps1)